simpleeval.py

```diff
@@ -102,6 +102,8 @@
 
 """
 
+__version__ = "1.0.1"
+
```

Comment:
> `import simpleeval;simpleeval.__version__`

This isn't a desirable incantation because it can run code. Instead, this is the best practice:

```
import importlib.metadata

importlib.metadata.version("simpleeval")
```

I recommend removing the version from the Python code.

Reply:
Seconded. When I as a prospective user of a package want to know whether it's installed, using a version that's compatible with my own code, etc., I use importlib and check its result *before* importing the package itself.

I know of a few libraries whose older versions I dare not import because they have ugly side effects … granted that simpleeval isn't and never was in that category, but best practices are there because you can use them without thinking about it.

Also as a personal preference I don't want to hard-code the version anywhere. The act of packaging, or releasing, shouldn't modify the package. Likewise, *not* packaging (as in `pip install -e .`) shouldn't make the package unusable and/or return stale or dummy data.

The release is tagged. That's your version right there. One source of truth is *way* better than three, any one of which can get out of sync with the others when you're not careful.

Reply:
@smurfix, I want to correct two things.

First, `pip install -e .` creates a `METADATA` file in a `$PACKAGE-$VERSION.dist-info/` directory in your `site-packages/` directory that contains the package version. `importlib.metadata.version()` reads the version from there, which means that the metadata can go stale.

```bash
# Linux
python -m venv venv-demo
source venv-demo/bin/activate
pip install -e .

ls -d venv-demo/lib/python*/site-packages/simpleeval-*.dist-info/
```

Second, 

> The release is tagged

This is good hygiene for the git repo. The version should still exist somewhere in the repo itself, and I advocate that it exist solely in `pyproject.toml`, not in Python code.

Reply:
Interesting.

I'm very happy to defer to this.

Thanks so much for the feedback!